decidim-admin/app/commands/decidim/admin/create_participatory_process_admin.rb

```diff
@@ -35,19 +35,39 @@ def call
 
       attr_reader :form, :participatory_process
 
-      def create_participatory_process_admin
-        ParticipatoryProcessUserRole.create!(
-          role: :admin,
-          user: user,
-          participatory_process: @participatory_process
-        )
+      def create_role
+        if form.roles.include?("collaborator")
+          user.roles << "collaborator" unless user.roles.include?("collaborator")
```

Comment:
I think a collaborator should be a refined participatory process admin with even less permissions. In that sense, I'd scope them to participatory processes.